locales/container/en.yml

```diff
@@ -5,6 +5,7 @@ en:
   advanced_settings:
     menu_order:
     - timezone
+    - dbbackup
```

Comment:
We have a separate backup solution for containers. We should probably re-consider the options in this list as a whole, but this one definitely doesn't belong.

Reply:
Butts... this was carry over from your last review, and I was cherry-picking in what it was in the previous diff:

https://github.com/ManageIQ/manageiq-appliance_console/compare/3f49d2de27e487e9d6b79d8261d45204be5e5281...NickLaMuro:rfe-db-dump-from-appliance-console-menu-orig

So I didn't take the time to look over what was reviewed there... sorry.  Will make the change.